documentation/Functions.md

```diff
@@ -176,10 +176,14 @@ cobiGenJava("path/to/java/file/MyEntity.java",[1,3,5,6,8])
 ***
 
 ### createDevon4jProject <a name="createDevon4jProject"></a>
+This function is used to create devon4j project.
```

Comment:
This function is used to create a [devon4j](https://github.com/devonfw/devon4j) project.